client/components/me-alerts/me-alerts.service.js

```diff
@@ -0,0 +1,73 @@
+class MeAlertsService {
+    constructor ($translate, $translatePartialLoader, CloudMessage, UserAlertsAapi, REDIRECT_URLS, TARGET) {
+        this.$translatePartialLoader = $translatePartialLoader.addPart("../components/me-alerts");
+        this.$translate = $translate;
+        this.CloudMessage = CloudMessage;
+        this.UserAlertsAapi = UserAlertsAapi;
+        this.REDIRECT_URLS = REDIRECT_URLS;
+        this.TARGET = TARGET;
+    }
+
+    getMessages () {
+        this.$translate.refresh().then(() => this.UserAlertsAapi.query({
```

Comment:
I'm curious about this $translate.refresh() thingy.. what does it accomplish?

Reply:
It's to make sure the translation tool is ready. That way we don't have "cloud_machin_chouette" labels across the manager. We used to have this issue a while ago.

Reply:
That's weird... the translator is initiated in a config and services are instantiated AFTER all config ran.